packages/babel-helpers/src/helpers.js

```diff
@@ -378,8 +378,12 @@ helpers.spreadByDefine = defineHelper(`
       var source = arguments[i];
       for (var key in source) {
         if (Object.prototype.hasOwnProperty.call(source, key)) {
-          var desc = Object.getOwnPropertyDescriptor(source, key);
-          Object.defineProperty(target, key, desc);
+          Object.defineProperty(target, key, {
```

Comment:
I fixed the previous (wrong) implementation, right now this should be what actually spread do according to the spec... as we mentioned him before, it would be awesome if @rauschma could doble check on this (thanks in advance for your patience 🙏 😌  ).

Reply:
It seems correct to me (just read the proposed spec)

Reply:
@nuragic Sorry, don’t have much time ATM. IIRC, the key thing with spread was: it reads properties via “get”, creates properties via “define”.

Reply:
@rauschma Cool, no worries! Sorry for bothering you… Thanks anyway, I think there's already a lot of detailed info in your blog post, plus all the code reviewers I've got are really the best ones! 😄